hubspot/management/commands/configure_hubspot_bridge.py

```diff
@@ -203,6 +203,11 @@
                 "targetHubspotProperty": "leadership_level",
                 "dataType": "STRING",
             },
+            {
+                "propertyName": "highest_education",
+                "targetHubspotProperty": "highest_education",
+                "dataType": "STRING",
+            },
```

Comment:
@tasawernawaz When this PR has been deployed to RC, the command `python manage.py configure_huspot_bridge` will need to be run on heroku - if you don't have heroku access I can do this for you.  It will also need to be run on production (by devops) when released.

Reply:
In order for this field to sync with hubspot, it will first need to be created as a custom property.  There is a `CUSTOM_ECOMMERCE_PROPERTIES` mapping in `hubspot.management.commands.configure_hubspot_bridge` that creates some new properties for deals.  A new section for "contacts" will need to be added to that, something like:
```javascript
"contacts": {
    "properties": [
        {
           "name": "highest_education",
           "label": "Highest Education",
           "description": "Highest education level",
           "type": "string",
           "fieldType": "text"
        }
     ]
}
```
https://developers.hubspot.com/docs/methods/contacts/create_property